src/main/resources/org/jenkinsci/plugins/workflow/job/WorkflowJob/configure-entries.properties

```diff
@@ -0,0 +1 @@
+pipeline.description=Define your Pipeline using Groovy directly or pull it from source control. Check the <a target="_blank" href="./pipeline-syntax">Pipeline Syntax</a> page for guidance on writing pipelines.
```

Comment:
A bit pedantic, but the `pipeline-syntax` link is actually defined in `workflow-cps` and specific to those `FlowDefinition` implementations. Can this blurb be kept in `workflow-cps`, and just shifted up in the `config.jelly`? Or can this aspect of the redesign be split out for separate consideration so we can go ahead with the rest, which certainly seems welcome?